packages/cozy-harvest-lib/src/components/KonnectorModal.jsx

```diff
@@ -10,17 +10,53 @@ import Modal, {
   ModalHeader
 } from 'cozy-ui/transpiled/react/Modal'
 import Spinner from 'cozy-ui/transpiled/react/Spinner'
-import { SubTitle } from 'cozy-ui/transpiled/react/Text'
 
 import accountMutations from '../connections/accounts'
 import triggersMutations from '../connections/triggers'
-import * as triggers from '../helpers/triggers'
+import * as triggersModel from '../helpers/triggers'
 import TriggerManager from './TriggerManager'
 import DeleteAccountCard from './cards/DeleteAccountCard'
 import LaunchTriggerCard from './cards/LaunchTriggerCard'
 import TriggerErrorInfo from './infos/TriggerErrorInfo'
 import withLocales from './hoc/withLocales'
+import SelectBox, {
+  reactSelectControl,
+  components
+} from 'cozy-ui/transpiled/react/SelectBox'
 
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import { Text } from 'cozy-ui/transpiled/react/Text'
+import { translate } from 'cozy-ui/transpiled/react/I18n'
+
+const MyAccountComponent = ({ name }) => {
+  return (
+    <Text className="u-slateGrey u-flex u-flex-items-center u-c-pointer">
+      {name} <Icon icon={'bottom-select'} size="12" className="u-ml-half" />
+    </Text>
+  )
+}
+
+const CreateAccount = translate()(({ createAction, t }) => {
+  return (
+    <Button
+      subtle
+      size={'small'}
+      className={'u-m-half'}
+      onClick={createAction}
+      label={t('modal.konnector.create_account')}
+    />
+  )
+})
+const MenuWithFixedComponent = props => {
+  const { children } = props
+  const { createAction, ...selectProps } = props.selectProps
```

Comment:
I would like to move this component to cozy-ui, but not sure how to handle the `createAction` prop. What do you think of having a prop called `fixedMenuProps` on the `Select` that is forwarded to the fixed menu element?

Reply:
Let's create an issue on UI to discuss about the API and component ? :D